src/Console/Command/Cache.php

```diff
@@ -52,15 +71,13 @@ public function handle()
     {
         $this->callSilent('route:clear');
 
-        $app = $this->getFreshApplication();
```

Comment:
Why would you remove this. 

```
$this->callSilent('route:clear'); 
```

This would clear the cache route, if you don't get a fresh application wouldn't `$app` already contained route from cached? This why even Laravel itself get a refresh application instance on route caching command.

https://github.com/laravel/framework/blob/5.7/src/Illuminate/Foundation/Console/RouteCacheCommand.php#L55

Reply:
This is at least my understanding but, it's a bit redundant precisely because Laravel already does this and caches the up-to-date routes, which is done by the 

```php
        $this->callSilent('route:clear');
```

Keen to hear from author though.

Reply:
I made this PR a long time ago and don't remember the details that accurately anymore.
Shortly afterwards we removed Dingo from our setup altogether (too little benefit for the performance cost).
In the meantime I've changed jobs and don't work with Laravel at all anymore.
Sorry I can't be of any more help anymore.

Reply:
Okey, I might just add that back just in case for now.